Geometry/RPCGeometryBuilder/src/RPCGeometryBuilderFromDDD.h

```diff
@@ -33,7 +33,7 @@ class RPCGeometryBuilderFromDDD
 
  private:
   RPCGeometry* buildGeometry(DDFilteredView& fview, const MuonDDDConstants& muonConstants);
-  std::map<RPCDetId,std::list<RPCRoll *> > chids;
+  std::map<RPCDetId,std::set<RPCRoll *> > chids;
```

Comment:
So this will re-order the rolls inside a chamber, changing ordering defined in DDD/DB to the raw Id index. 
Therefore the index mapping of some rolls are swapped in the validation module. 

There will be no cost to roll back to list. I can make a PR to fix it if you agree.